### DISCUSSION TITLE
Translation of example sentence for 羽振りがいい is incorrect

### DISCUSSION THREAD

**Discussion by Morgawr** [Jitendex/Jitendex#20] [Errors and Corrections]
Japanese:

> 彼女は昔羽振りが良かったに違いない。

English:

> She must have seen better days.

---

The meaning of 羽振りがいい basically means "to be influential" or "to have power", however I consulted with a native speaker and a more modern usage would be closer to (and I quote): "基本金持ちでお店とかでも気前よくお金を払うみたいな使われ方が多い。"

This said, I think the English example sentence in general is just misleading so it should be changed. I'd probably like to change it to something like:

> She must have been a very influential person in the past.

but I'm open to other interpretations/suggestions too, like:

> She must have been well-off in the past.

**Comment by stephenmk** [comment]
[Progressive J-E](https://dictionary.goo.ne.jp/word/en/%E7%BE%BD%E6%8C%AF%E3%82%8A/#je-61504) is the only reference I see that separates the "money" / "well-off" and "powerful" / "influential" meanings into distinct senses. Shinmeikai and daijirin mention 金力 alongside 権力. Seems like this term usually implies both at least to some extent.

I updated the meaning to your second suggestion. I think it's broad enough to cover either interpretation of the word.
